examples/stress_tests/time_smoothness.rs

```diff
@@ -0,0 +1,235 @@
+use bevy::{
+    diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
+    prelude::*,
+    window::{PresentMode, WindowMode},
+};
+
+// background is 905x363. To reduce empty space on each side and make sure
```

Comment:
The value given here doesn't match that below. Can we remove the reference to the size of the background from this comment to avoid further desync?